### PR TITLE
Bug 1690297, ose-ansible is only required for containerized installation

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1690297

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jianlinliu Please provide a QE review

File: install/disconnected_install.adoc [hunk 1]
```diff
@@ -184,6 +184,9 @@ Replace `<tag>` with the version to install. For example, specify `{latest-tag}`
 for the latest version. You can specify a different minor version.
 endif::[]
 +
+If you are using a containerized installation, pull
+`registry.redhat.io/openshift3/ose-ansible` in addition to these required images:
```
Comment (jianlinliu) [inline install/disconnected_install.adoc]: Should specify an exact major release tag for ose-ansible, such as: registry.redhat.io/openshift3/ose-ansible:v3.11

File: install/disconnected_install.adoc [hunk 2]
```diff
@@ -351,7 +353,9 @@ $ mkdir </path/to/images>
 $ cd </path/to/images>
 ----
 
-. Export the {product-title} infrastructure component images:
+. Export the {product-title} infrastructure component images. If you are using a
```
Comment (jianlinliu) [inline install/disconnected_install.adoc]: The same comments as the above.

**Comment by jianlinliu** [comment]
LGTM.

**Comment by ahardin-rh** [comment]
Applied in https://github.com/openshift/openshift-docs/pull/14265

**Pull request merged**